src/libs/index.ts

```diff
@@ -0,0 +1,4 @@
+import { errorHandler } from './routes/errorHandler';
```

Comment:
- First make index file inside routes folder and export errorHandler and notFoundRoute file from there
- Then import these file e.g., import { notFoundRoute, errorHandler } from './routes';